ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -73,5 +80,53 @@ public boolean isEnabled() {
 		public void setEnabled(boolean enabled) {
 			this.enabled = enabled;
 		}
+
+		public void setLingers_ms(Optional<Long> lingers_ms) {
```

Comment:
Why a set that takes an optional? Seems like you would just not set anything in that case this is an odd API.